proxy-node-gateway/src/main/resources/uk/gov/ida/notification/views/error-page.mustache

```diff
@@ -0,0 +1,50 @@
+<html>
+<head>
+    <title>Sorry, something went wrong</title>
+    <meta charset="utf-8"/>
+    <meta content="width=device-width,initial-scale=1.0" name="viewport"/>
+</head>
+<style>
+    .main {
+        color: rgb(51, 51, 51);
+        font-family: "Helvetica Neue", Arial, Helvetica, sans-serif;
+        text-align: center;
+        margin-top: 20px;
+        margin-bottom: 20px;
+    }
+    .logo {
+        /* maximum that displays in Chrome/macOS */
+        font-size: 7em;
+        margin-top: 20px;
+        margin-bottom: 20px;
+    }
+    .title {
+        font-weight: 700;
+        font-size: 3em;
+        margin-top: 20px;
+        margin-bottom: 20px;
+    }
+    .issues {
+        font-weight: 300;
+        margin-top: 20px;
+        margin-bottom: 20px;
+    }
+    .help {
+        margin-top: 20px;
+        margin-bottom: 20px;
+    }
+</style>
+<body class="main">
+<div class="logo">{{reaction}}</div>
```

Comment:
How do the variables get passed to the template? I can see where they're coming from in the ErrorPageView, but is there some magic that passes them to moustache?

Reply:
Moustache does some magic behind the scenes, it uses the getters on the `ErrorPageView` so for a variable called `reaction` it uses the getter `getReaction`.